src/pages/components/button/usage.mdx

```diff
@@ -102,186 +132,419 @@ import { Add16 } from "@carbon/icons-react";
+See [button groups](#Button-groups) below for more detailed information.
+
 <Row>
 <Column colLg={8}>
 
-![Secondary / Primary button alignment in forms](images/button-usage-7.png)
+![Example of a primary button matching the width of a card in a layout](images/button_usage_15_test.png)
 
-<Caption>Secondary / Primary button alignment in forms</Caption>
+<Caption>
+  Example of a primary button matching the width of a card in a layout
+</Caption>
 
 </Column>
 </Row>
 
-### Multistep forms
+### Button groups
 
-When the primary action implies a navigation step forward, as in a wizard, align single buttons or Secondary/Primary button groups to the right. This position conveys the “next step” intention.
+Button groups are a useful way of aligning buttons that have a relationship. Group the buttons logically into sets based on usage and importance. Too many calls to action will overwhelm and confuse users so they should be avoided.
+
+As mentioned in the [Emphasis section](#Emphasis), you don’t necessarily need to use the buttons in the order that their labels imply. Either a secondary or a tertiary button can be used in conjunction with a primary button. In fact, due to the visual weight of the secondary button, it’s recommended to use tertiary or ghost buttons in layouts with more than three calls to action.
 
 <Row>
 <Column colLg={8}>
 
-![Primary button alignment in a wizard](images/button-usage-8.png)
+![multiple buttons in a layout](images/button_usage_16.png)
 
-<Caption>Single button alignment in a wizard</Caption>
+</Column>
+</Row>
+
+<DoDontRow>
+  <DoDont caption="Do use a primary and two of the same lower emphasis buttons in a button group.">
+
+![Do use a primary and two of the same lower emphasis buttons in a button group.](images/button_usage_17.png)
+
+  </DoDont>
+  <DoDont type="dont" caption="Do not mix primary, secondary, and tertiary buttons in the same button group.">
+
+![Do not mix primary, secondary, and tertiary buttons in the same button group.](images/button_usage_18.png)
+
+  </DoDont>
+</DoDontRow>
+
+#### Button groups and the grid
+
+Ideally, when using groups of related buttons (not including ghost buttons), they should all be the same width. This can be achieved in one of two ways, both of which are acceptable. The first approach involves using the [narrow grid mode](/guidelines/2x-grid/code). In this situation each button would be set individually on the column grid.
+
+<Row>
+<Column colLg={8}>
+
+![Example of hanging buttons on the grid](images/button_usage_19.png)
+
+<Caption>Example of hanging buttons</Caption>
 
 </Column>
 </Row>
 
+Some products don’t want to hang their buttons into the gutter, but still want a 16px distance between buttons. This can be achieved while maintaining same-width buttons by treating the button group as a single object (rather than two separate entities) on the grid. Each button will essentially span 50% of their container with a programmatic 16px gutter between them.
```

Comment:
Buttons are currently implemented as spanning full-width, up to 320px. I think we could tie this section into the full bleed functionality we could add in. This "size" (or whatever terminology makes sense) would cause the button to span 100% of the container, and placing multiple of them in a row could create this behavior naturally. Could also add in a `ButtonGroup` component or something that helps with wrapping at the small breakpoint

Reply:
Back in the day working with @tw15egan on buttons, he mentioned that this was how same-size button groups should be implemented. This is probably the single most important update to button guidance for the design community in this PR. We've talked to Diana, Tessa, Erica, Tracey and Kaley about this update. They are already implementing this way on their end. Not sure if we can update the language here somehow, but this guidance is crucial to achieving same size-buttons groups, which is foundational to the system.